<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lernplanUI</class>
 <widget class="QDialog" name="lernplanUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lernplan</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDateEdit" name="EndDatum">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ende des Lernplans:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Loeschen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
 
  border-radius: 10px;
  padding: 0 8px;
;background-color: rgb(200, 24, 1);
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>LÃ¶schen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; text-decoration: underline;&quot;&gt;Lernplan erstellen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>-10</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 24, 1);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>-10</y>
     <width>74</width>
     <height>74</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
 
  border-radius: 10px;
  padding: 0 8px;
background-color: rgb(0, 121, 0);
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Erstellen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abbrechen">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>490</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">

QPushButton
{
 
  border-radius: 10px;
  padding: 0 8px;
background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Abbrechen</string>
   </property>
  </widget>
  <zorder>EndDatum</zorder>
  <zorder>label_2</zorder>
  <zorder>Loeschen</zorder>
  <zorder>label</zorder>
  <zorder>lbl2</zorder>
  <zorder>lbl1</zorder>
  <zorder>lbl3</zorder>
  <zorder>ok</zorder>
  <zorder>abbrechen</zorder>
  <zorder>label_12</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
